WordPress/Classes/ViewRelated/Blog/Blog Dashboard/Cards/Quick Actions/DashboardQuickActionsCardCell.swift

```diff
@@ -109,7 +109,7 @@ extension DashboardQuickActionsCardCell {
     }
 
     private func trackQuickActionsEvent(_ event: WPAnalyticsStat, blog: Blog) {
-        WPAppAnalytics.track(event, withProperties: [WPAppAnalyticsKeyTapSource: "dashboard"], with: blog)
+        WPAppAnalytics.track(event, withProperties: ["tab_source": "dashboard", "tap_source": "quick_actions"], with: blog)
```

Comment:
[nit] What about reverting back to using `WPAppAnalyticsKeyTapSource` and adding a new constant for "tab_source" as well? 
This could help in avoiding typos in the future.

Reply:
Good tip! Done in c0d568301001fda8a4ff5894accd00ae6c44fed1 and 14254d0473444f18c99665b7c206b30f1e0625ed